### PR TITLE
fix: header responsivenes

### PR DESCRIPTION
**Pull request by Ismoil-JS**

### CODE REVIEW

File: src/assets/logo-full.svg
Comment (purple-mountain) [inline src/assets/logo-full.svg:1]: is it possible to add a bit extra space between the logo and logo text? also, can you delete "stays and experiences" completely? If I am not mistaken, it is not used anywhere and is not needed.

Comment (purple-mountain) [reply src/assets/logo-full.svg:1]: can you also add some padding or margin between the category part and the search part? on mobile they kind of start blending into each other: 

image

**Review by purple-mountain — APPROVED** [review]

**Comment by purple-mountain** [comment]
by the way, I just noticed but the ticket number is duplicate. it should be 63.

File: src/assets/logo-full.svg
Comment (Ismoil-JS) [reply src/assets/logo-full.svg:1]: added padding, space added on logo, now I do not want to delete it 'stay and experience', we may add that functionality also

**Comment by Ismoil-JS** [comment]
> by the way, I just noticed but the ticket number is duplicate. it should be 63. I updated it in trello

**Pull request merged**